src/Microsoft.IdentityModel.Tokens/Base64UrlEncoder.cs

```diff
@@ -152,8 +152,7 @@ public static string Encode(byte[] inArray, int offset, int length)
         }
 
         /// <summary>
-        /// Converts a subset of an array of 8-bit unsigned integers to its equivalent string representation that is encoded with base-64-url digits. Parameters specify
-        /// the subset as an offset in the input array, and the number of elements in the array to convert.
+        /// Converts a subset of an array of 8-bit unsigned integers to its equivalent string representation that is encoded with base-64-url digits.
```

Comment:
nit: grammar, I *think* it should actually be "representation which is encoded" as it's non-definitive and merely an additional explanation.